code/modules/mob/living/living.dm

```diff
@@ -1093,7 +1093,8 @@
 	var/stun = IsStun()
 	var/knockdown = IsKnockdown()
 	var/ignore_legs = get_leg_ignore()
-	var/canmove = !IsImmobilized() && !stun && conscious && !paralyzed && !buckled && (!stat_softcrit || !pulledby) && !chokehold && !IsFrozen() && (has_arms || ignore_legs || has_legs)
+	var/in_stasis = IsInStasis()
+	var/canmove = !IsImmobilized() && !stun && conscious && !paralyzed && !buckled && (!stat_softcrit || !pulledby) && !chokehold && !IsFrozen() && !in_stasis && (has_arms || ignore_legs || has_legs)
```

Comment:
yikes that's a mouthful

Reply:
Would you like it to be more vertical? Like
```cpp
	var/canmove = !IsImmobilized() && !stun && conscious && \
	!paralyzed && !buckled && (!stat_softcrit || !pulledby) && \
	!chokehold && !IsFrozen() && !in_stasis && \
	(has_arms || ignore_legs || has_legs)
```

Reply:
it's fine but wowzers lmao